datadog_lambda/wrapper.py

```diff
@@ -248,7 +243,11 @@ def __call__(self, event, context, **kwargs):
             self.response = self.func(event, context, **kwargs)
             return self.response
         except Exception:
-            submit_errors_metric(context)
+            if not should_use_extension:
+                from datadog_lambda.metric import submit_invocations_metric
+
+                submit_invocations_metric(context)
```

Comment:
I don't think we want this change. You replaced an errors metric with an invocation metric. 😄

Reply:
Woops, thanks for catching that!